js/src/admin/components/WebhookEditModal.js

```diff
@@ -148,7 +152,8 @@ export default class WebhookEditModal extends Modal {
                       <div>
                         <h3>{this.translate(group)}</h3>
                         {events.map((event) => (
-                          <Switch state={this.webhook.events().includes(event.full)} onchange={this.onchange.bind(this, event.full)}>
+                          <Switch state={this.webhook.events().includes(event.full)}
```

Comment:
Formatting changes should be left out. There are a few reasons for that, but the most important one is that it makes the review process so much harder. You really want to just submit the changes that have an actual impact. Especially in open source where you want to make the impact of reviewal as non-existent as possible.

Understand that you could have submitted these changes unknowingly, most IDE's have settings that enable auto formatting before commit. It's best to disable this feature and use the keyboard shortcut to make this a manual action. If per-project settings are possible, then doing that would be an alternative.